scripts/errors/generate-errors.js

```diff
@@ -0,0 +1,37 @@
+const fs = require("fs");
```

Comment:
This js code will need to be transformed into golang code.

Reply:
Plan is to still call this script but have template in go binary. 

@craicoverflow if you have concerns about that workflow/scope let me know

Reply:
Not sure what you mean. So plan is to call this Node.js script from the Go executable?

Reply:
My final version runs internal/apigen/api-generate-errors-all/main.go  binary instead of generate-errors.js.